content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -182,8 +182,8 @@ The control plane mirrors Endpoints resources unless:
   set to `true`.
 * the Endpoints resource has a `control-plane.alpha.kubernetes.io/leader`
   annotation.
-* the corresponding Service resource does not exist.
-* the corresponding Service resource has a non-nil selector.
+* the Endpoints resource does not exist.
```

Comment:
This doesn't make sense. If you have an Endpoints, it will be mirrored. If you don't have an Endpoints, the control plane correctly doesn't make any EndpointSlices.